manifests/repo/el.pp

```diff
@@ -26,4 +28,11 @@
       enabled  => 1,
     }
   }
+
+  if $yum_proxy {
+    Yumrepo['jenkins'] {
+      proxy => $yum_proxy,
+    }
```

Comment:
Does this need to be done as a resource default?  I feel that they are somewhat of an anti-pattern until if/when the scope is restricted.  I've battled endless problems with defaults leaking out and causing unintended side-effects.